packages/earn-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.0]
+
+### Changed
+
+- Updated `EarnController` methods (`refreshPooledStakingData`, `refreshPooledStakes`, and `refreshStakingEligibility`) to use an options bag parameter ([#5537](https://github.com/MetaMask/core/pull/5537))
+- Updated `AccountsController:selectedAccountChange` subscription handler to pass `account?.address` from the event payload into the underlying methods ([#5537](https://github.com/MetaMask/core/pull/5537))
```

Comment:
Sorry I just had another look at your PR. we don't need to include this CHANGELOG entry